docs/release-notes/november-2023.md

```diff
@@ -0,0 +1,138 @@
+title: November 2023
+tags:
+  - Release notes
+---
+
+# November 2023 release notes
+
+:::tip
+- **Semgrep Pro Engine** is now generally available (GA). Team tier users and above can use the Pro Engine to perform **cross-file (interfile)** and **cross-function (intrafile)** analyses. To enable Semgrep Pro Engine:
+    1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login)
+    1. Click **Settings**.
+    1. Click the **<i class="fa-solid fa-toggle-large-on"></i> Semgrep Pro Engine** toggle.
+- See [<i class="fa-regular fa-file-lines"></i> Semgrep Pro Engine](/semgrep-code/semgrep-pro-engine-intro/) documentation for more information.
+- The Semgrep command-line tool now requires **Python 3.8** or later.
+:::
+
+## 🔧 Semgrep OSS Engine
+
+:::note
+Beginning with version 1.46.0, Semgrep is first released to:
+- `pypy`
+- `brew`
+- `returntocorp/semgrep:canary` (Docker)
+
+If no issues are detected after a few days, the Semgrep team then promotes the `:canary` Docker tag to `:latest`.
+:::
+
+- The following versions of Semgrep OSS Engine were released in November 2023:
+  - [<i class="fas fa-external-link fa-xs"></i> 1.47.0](https://github.com/returntocorp/semgrep/releases/tag/v1.47.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.48.0](https://github.com/returntocorp/semgrep/releases/tag/v1.48.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.49.0](https://github.com/returntocorp/semgrep/releases/tag/v1.49.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.50.0](https://github.com/returntocorp/semgrep/releases/tag/v1.50.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.51.0](https://github.com/returntocorp/semgrep/releases/tag/v1.51.0)
+
+
+## 🌐 Semgrep Cloud Platform
+
+### Added
+
+- Semgrep now records the languages using interfile analysis during a scan. This enables the Semgrep team to measure new Pro Engine languages' performance impact and error rates. **For scans that don't send metrics, there is no change.** See [<i class="fas fa-external-link fa-xs"></i> Semgrep Privacy Policy](https://github.com/semgrep/semgrep/blob/develop/PRIVACY.md) for more information.
+- Added a link to the SSO documentation to help users set up SSO. <!-- 11485 -->
+- **CLI tool:** Added `--config code` and `--config secrets` flags to the `semgrep scan` command. When using these flags, the environment variable SEMGREP_REPO_NAME must be set. For example,
+    ```
+    $ SEMGREP_REPO_NAME=test_repo semgrep --config secrets
+    ```
+
+### Changed
+
+- Elixir language support now requires the Pro Engine. To scan Elixir codebases, enable the Pro Engine. <!-- 9308 -->
+- The Semgrep CLI tool now correctly counts the rules run on a codebase. Previously, Semgrep counted the total rules in the user's Policies or rulesets, including rules that did **not** have valid targets and therefore, did not actually run. <!-- 9130  -->
+- Updated instances of **returntocorp** to **Semgrep**. <!-- gh 112877 -->
```

Comment:
we probably don't need to include this. I'm guessing this is mostly our github name changes

Reply:
We are erring on the side of caution + extra transparency partially because of [feedback like this on the OSS product](https://github.com/semgrep/semgrep/issues/8437).